Rename updated_after to updated_since

WouterSioen opened the pull request:
Note that we'll still support updated_after for a while, but will
remove it later.

maartenscholz commented:
No, good catch

luisabritotl commented on apiary.apib:
```diff
@@ -1649,6 +1649,110 @@ Delete a meeting from Teamleader
+        + due_date: 1265412302 (number, required) - Due date of the task as Unix timestamp
+        + `user_id`: 1 (number) - Id of the user (is required if team_id is not used)
+        + `team_id`: 2 (number) - Id of the team (is required if user_id is not used)
+        + task_type_id: 3 (number, required) - Category of the task
```
PP: change task_category or change the description to Type of tasks

maartenscholz replied:
I just copied the description from the old docs but I can update if that's what we are doing

luisabritotl commented on apiary.apib:
```diff
@@ -1649,6 +1649,110 @@ Delete a meeting from Teamleader
+        + `user_id`: 1 (number) - Id of the user (is required if team_id is not used)
+        + `team_id`: 2 (number) - Id of the team (is required if user_id is not used)
+        + task_type_id: 3 (number, required) - Category of the task
+        + duration: 120 (number, required) - Amount of mintues that should be spent on the task
```
minutes

luisabritotl commented on apiary.apib:
```diff
@@ -1649,6 +1649,110 @@ Delete a meeting from Teamleader
+                + project_milestone
+        + for_id: 4 (number) - The id of who this task should be billed to
+        + creator_user_id: 5 (number) - Id the user that created the task
+        + related_deal_id: 6 (number) - Id of the deal related to the task
```
is the word related necessary?

maartenscholz replied:
Yeah, `deal to the task` is not really correct english in this context

luisabritotl commented on apiary.apib:
```diff
@@ -1649,6 +1649,110 @@ Delete a meeting from Teamleader
+    + Attributes
+        + task_id: 1 (number, required) - Id of the task
+        + description: Description (string) - Description of the task
+        + duration: 120 (number) - Amount of mintues that should be spent on the task
```
minutes

luisabritotl commented on apiary.apib:
```diff
@@ -1649,6 +1649,110 @@ Delete a meeting from Teamleader
+        + amount: `25` (number, required) - The amount of tasks returned per request
+        + pageno: `0` (number, required) - The current page
+        + selected_customfields: `42,142` (string) - Comma-separated list of the Id's of the custom fields you wish to select
+        + user_id: `1` (number) - Rreturn only tasks for this user
```
Return

WouterSioen committed:
Rename updated_after to updated_since

Note that we'll still support updated_after for a while, but will
remove it later.

WouterSioen commented:
@mrtus @DavidTeamleader Do you already use this filter? And if so, can you migrate this asap so we can remove the deprecated `updated_after` filter?

DavidTeamleader commented:
@WouterSioen We don't use this filter at the moment, so won't have to do any migration :)
